mantis-control-plane/mantis-control-plane-dynamodb/src/test/java/io/mantisrx/extensions/dynamodb/DynamoDBMasterMonitorSingletonTest.java

```diff
@@ -133,13 +138,13 @@ public void monitorDoesNotReturnNull() throws IOException, InterruptedException
             .atLeast(DynamoDBLockSupportRule.heartbeatDuration)
             .pollDelay(DynamoDBLockSupportRule.heartbeatDuration)
             .atMost(Duration.ofMillis(DynamoDBLockSupportRule.heartbeatDuration.toMillis()*2))
-            .untilAsserted(() -> assertEquals(MasterDescription.MASTER_NULL, m.getLatestMaster()));
+            .untilAsserted(() -> assertEquals(MasterDescription.MASTER_NULL, m.getMasterSubject().getValue()));
         lockSupport.releaseLock(lockKey);
 
         m.shutdown();
 
-        testSubscriber.assertNoTerminalEvent();
-        testSubscriber.assertNotCompleted();
+        testSubscriber.awaitTerminalEvent(10, TimeUnit.SECONDS);
+        testSubscriber.assertCompleted();
```

Comment:
We do want the observable to complete so that subscribers can do any necessary clean up.

Reply:
That tracks.